src/allmydata/web/storage.py

```diff
@@ -16,91 +17,108 @@ def remove_prefix(s, prefix):
     return s[len(prefix):]
 
 
-class StorageStatus(MultiFormatPage):
-    docFactory = getxmlfile("storage_status.xhtml")
-    # the default 'data' argument is the StorageServer instance
+class StorageStatusElement(Element):
+    loader = XMLFile(FilePath(__file__).sibling("storage_status.xhtml"))
 
-    def __init__(self, storage, nickname=""):
-        rend.Page.__init__(self, storage)
+    def __init__(self, storage, nickname):
```

Comment:
Added a docstring in dc9b1b2, although I suppose it could be improved:

```python
def __init__(self, storage, nickname=""):
        """
        :param _StorageServer storage: data about storage.
        :param string nickname: friendly name for storage.
        """
```